vue-frontend/src/components/home/HowDoWeDiffer.vue

```diff
@@ -302,5 +296,24 @@ export default {
   .flex-grid-2 {
     justify-content: center;
   }
+
+  .description {
+    padding-top: 112px;
+  }
+}
+
+@include media-breakpoint-up(xl) {
+  .how-do-we-differ-section {
+    width: 60%;
+    min-width: 900px;
+  }
+
+  .differ-description {
+    padding: 0 10%;
+  }
+
+  .description {
+    padding-top: 116px;
```

Comment:
Should we use static paddings?

Reply:
Because override the section in above components so that used static padding

Reply:
Can't we apply in percentage?